README.md

```diff
@@ -824,6 +825,228 @@ and Equinox will supply the _initial_ value for the `project` function to render
+
+> NOTE: There are more advanced forms that allow the `decide` function to be `Async` and/or to also return a `'result`, which will be yielded to the caller driving the Decision as the return value of the `Transact` function.
+
+#### So what kind of a thing is a Decider then?
+
+Is it a marketing term? Jérémie's way of explaining an app?
+
+I'd present the fact that Equinox:
+- was initially generalised and extracted from working code using ESDB in (by most measures) a successful startup written in a mix of F# and C# by the usual mix of devs, experience levels, awareness of event sourcing patterns and language/domain backgrounds
+- for a long time only had its MemoryStore as the thing to force it to be store agnostic
+- did not fundamentally change to add idiomatic support for a Document database (CosmosDB)
+- will not fundamentally change to add idiomatic support for DynamoDB
+- can and has been used at every stage in an e-commerce pipeline
+- is presently aligning pretty neatly with diverse domains without any changes/extensions, both for me and others
+
+... as evidence for Decider being *a pattern language* (analogous to how event sourcing and many modern event sourced UI paradigms share a lot of common patterns).
+
+#### ... about the process of making _decisions_
+
+Finally, I'd say that a key thing the Decider concept brings is a richer way of looking at event sourcing than the typical event sourcing 101 examples you might see:
+- de-emphasizing one-way things that map commands to events without deduplicating and/or yielding a result (not saying that you shouldn't do the simplest thing -- you absolutely should)
+- de-emphasizing all projection handlers only ever just sitting there looking for `MyThingCreated` and doing an `INSERT` with a try/catch for duplicate inserts as a way to make it idempotent (and every stream design requiring a Created event as a way to enable that)
+
+The missing part beyond that basic anemic stuff is where the value lies:
+- any interesting system *makes _decisions_*:
+- a decision can yield a result alongside the events that are needed to manifest the state change
+- any decision process can and should consider [idempotency](https://en.wikipedia.org/wiki/Idempotence) - if you initiate a process/request something, a retry can't be a special case you don't worry about, considering it is a fundamental baseline requirement and thinking tool
+- it can let you drive a set of reactions in a fault-tolerant and scalable (both perf/system size, and management/separation of complexity)  manner
+
+Quite frequently, a Decider will also internally be a Process Manager, encapsulting a state machine.
+
+### With `Equinox.CosmosStore`, it seems it should be possible to handle saving multiple events from multiple streams as long as they share the same partition key in Cosmos DB. But it does not seem to be possible via `Equinox.Decider.Transact` ? :pray: [@rmaziarka](https://github.com/rmaziarka)
```

Comment:
**QUESTION:** Does all strategies for different stores are always really transactional? Also, it'd be clear to say what do you mean by Transactional. Is it really a transaction or more atomicity?

Reply:
hopefully rewording improved it. this was definitely lazy